docs/language/values-and-types.mdx

```diff
@@ -1076,6 +1076,27 @@ are available for both variable-sized and fixed-sized or variable-sized arrays.
   let containsKitty = numbers.contains("Kitty")
   ```
 
+- `cadence•fun slice(from: Int, upTo: Int): [T]`
+
+  Returns an array slice of the elements
+  in the given array from start index `from` up to,
+  but not including, the end index `upTo`.
+  This function creates a new array whose length is `upTo - from`.
+  It does not modify the original array.
+  If either of the parameters are out of
+  the bounds of the array, the function will fail.
```

Comment:
What if `upTo < from`? This should probably fail in some way, but we should specify the behavior. This case also appears unspecified in the `slice` function for strings as well.

Reply:
Good question! Clarified in the docstrings and documentation in ce4487eb2e16b444667ea9d654407ff3de22b376